raft/src/main/java/org/apache/kafka/raft/internals/BatchAccumulator.java

```diff
@@ -256,7 +256,7 @@ public void appendControlMessages(MemoryRecordsCreator valueCreator) {
     }
 
     private int validateMemoryRecordAndReturnCount(MemoryRecords memoryRecord) {
-        // Confirm that it is at most one batch and it is a control record
+        // Confirm that it is one control batch and it is at least one control record
```

Comment:
validateMemoryRecordAndReturnCount => validateMemoryRecordsAndReturnCount
memoryRecord => memoryRecords
Also, there is an existing typo creatte on line 268.

Reply:
Fixed.